dist/images/install.sh

```diff
@@ -3408,6 +3409,23 @@ reload(){
   kubectl rollout status deployment/kube-ovn-monitor -n kube-system
 }
 
+env-check(){
+  set +e
+  echo "************************************************************************"
+  echo "The result will be saved to /var/log/kube-ovn/env-check.log on each node"
+  echo "************************************************************************"
+
+  KUBE_OVN_NS=kube-system
+  podNames=`kubectl get pod -n $KUBE_OVN_NS | grep kube-ovn-cni | awk '{print $1}'`
```

Comment:
`--no-headers` should be added to kubectl command.

Reply:
modified